support-frontend/assets/helpers/page/page.js

```diff
@@ -82,10 +83,20 @@ function buildInitialState(
 ): CommonState {
   const excludedParameters = ['REFPVID', 'INTCMP', 'acquisitionData'];
   const otherQueryParams = getAllQueryParamsWithExclusions(excludedParameters);
-  const localCurrencyCountry = localCurrencyCountries[
+  let localCurrencyCountry;
```

Comment:
it would be nice to call a function here instead, e.g.
```
function getLocalCurrencyCountry(
  countryId: IsoCountry,
  abParticipations: Participations
): ?LocalCurrencyCountry {
  const queryParam = getQueryParameter('local-currency-country');
  if (queryParam) {
    return localCurrencyCountries[queryParam.toUpperCase()];
  }

  if (abParticipations.localCurrencyTest === 'variant') {
    return localCurrencyCountries[countryId];
  }

  return null;
}
```

Reply:
Much better!